README.md

```diff
@@ -164,8 +169,43 @@ Kiwix-build.py will create several directories:
 If you want to install all those directories elsewhere, you can pass the
 `--working-dir` option to `kiwix-build`:
 
-CI/CD containerised environments
---------------------------------
+Kiwix CI/CD
+-----------
+
+Kiwix Build workflow produces a few different things necessary to the
+Kiwix CI/CD.
+
+#### Base dependences
+
+Base dependences are tarballs where third party dependences are
+pre-compiled. The purpose of this is to speed-up Kiwix Build own
+CI. They are available [here](http://tmp.kiwix.org/ci/).
+
+Each time we use the CI/CD of Kiwix Build, it is checked if an
+appropriate base dependences tarball is available. If "yes" it is
+downloaded and used, otherwise it will be built and uploaded.
+
+If you need to create a new tarball you can either:
+* Increment the version number of them in `kiwixbuild/versions.py` a
+  new version is uploaded
+* Delete the multiple tarballs related to current version in
+  tmp.kiwix.org/ci.
```

Comment:
Preferably the first option. Deleting tarballs on ftp should be a last solution (and done only by us)